tools/lxdclient/utils.go

```diff
@@ -65,6 +66,8 @@ func IsRunningLocally() (bool, error) {
 	return running, nil
 }
 
+const errIPV6NotSupported = `socket: address family not supported by protocol`
```

Comment:
a comment here about how we arrived at this string would be useful.
did you trace where the actual string was generated to know if it is stable?